paths_cli/tests/compiling/test_tools.py

```diff
@@ -13,6 +14,13 @@
 def test_custom_eval(expr, expected):
     npt.assert_allclose(custom_eval(expr), expected)
 
+def test_custom_eval_int():
+    assert custom_eval_int('5') == 5
+
+def test_custom_eval_int_strict_pos_error():
+    with pytest.raises(InputError):
+        custom_eval_int_strict_pos(-1)
```

Comment:
just my paranoia, but do you mind adding the explicit test for `0` as well?
```suggestion
    with pytest.raises(InputError):
        custom_eval_int_strict_pos(-1)
    with pytest.raises(InputError):
        custom_eval_int_strict_pos(0)
```

Reply:
Added as `parametrized`.